sklearn/preprocessing/data.py

```diff
@@ -1730,13 +1728,14 @@ def _transform_selected(X, transform, selected="all", copy=True):
 
 
 class OneHotEncoder(BaseEstimator, TransformerMixin):
-    """Encode categorical integer features using a one-hot aka one-of-K scheme.
+    """Encode ordinal integer features using a one-hot aka one-of-K scheme.
```

Comment:
I would say categorical and in the next line clarify they need to be 0 to (n_categories - 1).